tensorflow/compiler/xla/refcounting_hash_map.h

```diff
@@ -22,6 +22,7 @@ limitations under the License.
 #include "absl/container/node_hash_map.h"
 #include "absl/memory/memory.h"
 #include "absl/synchronization/mutex.h"
+#include "tensorflow/core/platform/logging.h"
```

Comment:
Is this required?

Reply:
this may not be required...will remove it